graphx/src/main/scala/org/apache/spark/graphx/lib/StronglyConnectedComponents.scala

```diff
@@ -36,7 +36,7 @@ object StronglyConnectedComponents {
    * @return a graph with vertex attributes containing the smallest vertex id in each SCC
    */
   def run[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED], numIter: Int): Graph[VertexId, ED] = {
-
+    require(numIter > 0)
```

Comment:
I think you'd need to add an error message, e.g.

s"Number of iterations ($numIter) must be greater than 0."